block_util/src/fixed_vhd_async.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright © 2021 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
```

Comment:
New code should be Apache 2.0 unless you copied from code that is derived from elsewhere.

Reply:
Ok I'll apply to all new files!